src/actions/notifications.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/prefer-default-export */
+import * as types from './types';
+import { axiosInstance } from '@Utils/';
+
+
+export const getNotifications = () => async dispatch => {
+  try {
+    const response = await axiosInstance.get('notifications/');
+    dispatch({
+      type: types.FETCH_NOTIFICATIONS,
+      payload: response.data.notifications,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    // console.log(err);
+  }
+};
+
+export const markAsRead = (id) => async dispatch => {
+  try {
+    const response = await axiosInstance.patch(`notifications/${id}/read`);
+    dispatch({
+      type: types.UPDATE_NOTIFICATIONS,
+      payload: response.data,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    // console.log(err);
+  }
+};
+
+export const markAllRead = () => async dispatch => {
+  try {
+    const response = await axiosInstance.patch('notifications/read');
+    dispatch({
+      type: types.MARK_READ,
+      payload: response.data,
+    });
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    // console.log(err);
```

Comment:
Nice work Jude. I like the UI design. I am just thinking, if you are not expecting an error in this API call, is there a need for using the try and catch block?

Reply:
Yes, I wrapped in a try and catch block to avoid the App from crashing in the advent of an error.

Reply:
Alright. Nice